### PR TITLE
Fix lint erros in image pkg

### PR DESCRIPTION
**Pull request by j-griffith**
Fix golint errors in image pkg

### CODE REVIEW

**Comment by j-griffith** [comment]
These errors seem to be intermittent, doing an empty re base to test that theory.

File: pkg/image/filefmt.go [hunk 1]
```diff
@@ -67,10 +67,12 @@ func CopyKnownHdrs() Headers {
 	return m
 }
 
+// Match performs a check to see if the provided byte array matches the bytes in our header data
```
Comment (jeffvance) [inline pkg/image/filefmt.go]: nit: technically byte slice not array...

File: pkg/image/filefmt.go [hunk 2]
```diff
@@ -67,10 +67,12 @@ func CopyKnownHdrs() Headers {
...
 func (h Header) Match(b []byte) bool {
 	return bytes.Equal(b[h.mgOffset:h.mgOffset+len(h.magicNumber)], h.magicNumber)
 }
 
+// Size uses the Header receiver and the provided byte array to extract the size or the original file represented in the headers metadata
```
Comment (jeffvance) [inline pkg/image/filefmt.go]: a bit more accurate would be:
```
// Size uses the Header receiver offset and length fields to extract, from the passed-in file header slice (b), the size of the original file.
```

File: pkg/image/validate.go [hunk 3]
```diff
@@ -1,12 +1,20 @@
 package image
 
 const (
-	ExtImg   = ".img"
-	ExtIso   = ".iso"
-	ExtGz    = ".gz"
+	// ExtImg is a constant for the .img extenstion
+	ExtImg = ".img"
+	// ExtIso is a constant for the .img extenstion
```
Comment (jeffvance) [inline pkg/image/validate.go]: copy/paste errors from here to bottom...

File: pkg/image/filefmt.go (hunk 1 above)
Comment (j-griffith) [reply pkg/image/filefmt.go]: done

**Comment by jeffvance** [comment]
lgtm
Waiting for ci

**Pull request merged**